package.json

```diff
@@ -21,6 +21,7 @@
     "node-sass": "^4.14.1",
     "react": "^16.13.1",
     "react-bootstrap": "^1.3.0",
+    "react-bootstrap-country-select": "^1.0.0",
```

Comment:
Let's get rid of changes to package.json and package-lock.json since you didn't end up needing these packages to give you country and state selection

Reply:
@bestfoodalex  removed and commits pushed to the branch!